android/sdk/src/main/java/com/taobao/weex/adapter/ClassLoaderAdapter.java

```diff
@@ -52,4 +52,5 @@
         }
     }
 
-}
\ No newline at end of file
+}
```

Comment:
Please don't commit this file

Reply:
just for android lint test.
I will remove it later